README.adoc

```diff
@@ -334,18 +334,19 @@ end
 
 # good
 describe '#attributes' do
-  subject { FactoryBot.create(:article) }
+  let(:article) { FactoryBot.create(:article) }
+  subject { article.attributes }
 
   specify do
-    expect(subject.attributes).to include subject.display_name
-    expect(subject.attributes).to include subject.created_at
+    expect(subject).to include article.display_name
```

Comment:
Isn't `attributes` a Hash?
Both the old and the new examples would fail.
In general, I'd write `expect(article).to have_attributes(display_name: 'something')`.
But the point of this is to demonstrate that different contexts should have different ... contexts.

Reply:
`Hash` [is compatible with include](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/include-matcher), but what I've got here doesn't achieve the desired result - you're right. This would work, though:

```suggestion
    expect(subject).to include(display_name: article.display_name)
```